docs/architecture/modernize-desktop/deploying-modern-desktop-applications.md

```diff
@@ -0,0 +1,332 @@
+quickly while other parts of the app are downloaded on the background.
+This feature contributes to an engaging experience for your users.
+
+Through optional packages feature you achieve componentization on your app
+deployment, so you can download them when needed.
+
+#### Simple packaging and deployment
+
+The AppManifest declares the versioning, device targeting and identify in a
+standard way for every application. It also provides a way to sign your assets
+providing a solid security foundation.
+
+#### OS Managed
+
+The OS handles all the processes for installation, updating and removing an
+application. Applications are installed per user but downloaded only once,
+minimizing the disk footprint. Microsoft is working on providing the MSIX
+experience also on Windows 7.
+
+#### Windows provides integrity for the app
+
+With the use of digital signatures, you can guarantee that you don't install
+application from untrusted sources. MSIX also prevents tampering because it
+keeps a record of file hashes and detects on file has been modified after
+installation.
+
+#### Works for the entire App Catalog
+
+One of the coolest things about MSIX is that it works for the entire application
+catalog, Windows Forms, WPF, MFC/ATL, Delphi, even if you want to do xCopy
+deployment, ClickOnce, or going to the Store, you can use the same MSIX package.
+
+### Tools
+
+#### Windows Application Packaging Project
+
+You can use the **Windows Application Packaging Project** project in Visual
+Studio to generate a package for your desktop app. Then, you can publish that
+package to the Microsoft Store or sideload it onto one or more PCs.
+
+#### MSIX Packaging Tool
+
+The MSIX Packaging Tool enables you to repackage your existing Win32
+applications to the MSIX format. It offers both an interactive UI and a command
+line for conversions and gives you the ability to convert an application without
+having the source code.
+
+![MSIX Packaging Tool](media\deploying-modern-desktop-applications\msix-packaging-tool.png)
+
+#### Package Support Framework
+
+The Package Support Framework is an open-source kit that helps you apply fixes
+to your existing win32 application when you don't have access to the source
+code, so that it can run in an MSIX container. The Package Support Framework
+helps your application follow the best practices of the modern runtime
+environment.
+
+#### App Installer
+
+App Installer allows for Windows 10 apps to be installed by double-clicking the
+app package. This means that users don't need to use PowerShell or other
+developer tools to deploy Windows 10 apps. The App Installer can also install an
+app from the web, optional packages, and related sets.
+
+## How to create an MSIX package from an existing Win32 desktop application
+
+Let's see the steps you have to take to create an MSIX package from an existing
+Win32 application, a Windows Forms app in this case.
+
```

Comment:
I guess this is skipping the step of opening your existing WinForms project in Visual Studio first. 
Also, the steps should be reviewed to show exactly where customers need to click or what they need to select.

1. Open your existing WinForms project in Visual Studio.
2. In Solution Explorer, right click on the solution node and select **Add** > **New Project**. The **Add New Project** dialog opens.
3. ...

And so on...